### PR TITLE
new list button

### PR DESCRIPTION
**Pull request by wsf246**
**Problem:**
Create a list button wasn't obvious
**Solution:**
Make a pinterest-esque new list button

### CODE REVIEW

**Comment by wsf246** [comment]
image

**Comment by briankung** [comment]
It's like...comically cartoonish XD but it looks fine

**Comment by briankung** [comment]
Hey, while we're here, can we make the background an off-white?

We could use `$gray-lighter` from https://github.com/twbs/bootstrap-sass/blob/master/assets/stylesheets/bootstrap/_variables.scss#L16

**Comment by wsf246** [comment]
we can use whichever glyphicon
http://getbootstrap.com/components/

I figure it has to be the pencil or the plus sign.  I'm fine w/ whatever

File: app/views/layouts/application.html.erb [hunk 1]
```diff
@@ -24,7 +24,7 @@
     <%= render 'application/favicon' %>
   </head>
 
-  <body data-spy="scroll" data-target=".scrollspy">
+  <body data-spy="scroll" data-target=".scrollspy" class= "<%= 'list-index' if (action_name == 'index' && controller_name == 'lists')%>">
```
Comment (briankung) [inline app/views/layouts/application.html.erb]: I'm not a huge fan of this - I'd prefer a `yield(:body_class)` and a `content_for(:body_class)` in index. Or some other way of doing it - you could wrap `list#index` in another div with the `list-index` class.

**Comment by wsf246** [comment]
cool, i used content_for instead.  we've also been using the @title and @full_width convention.  Do you have preference between that and content_for?

**Comment by wsf246** [comment]
are the quotes strictly necessary?  it seems to work without them as well.

**Comment by briankung** [comment]
No, they aren't, because HTML was made out of 💩 for thousands of years.

We should definitely have quotation marks around class names.

**Comment by wsf246** [comment]
:+1:  good to merge?

**Pull request merged**